src/repo.rs

```diff
@@ -85,12 +100,12 @@ impl Repo {
 
         fs::write(&full_path, toml_string)?;
 
-        info!("Commiting with message \"{}\".", commit_msg);
-        git::add_file(&self.path, &*self.config.repo.file, false)?;
-        git::commit_all(&self.path, &*commit_msg, false, false)?;
+        println!("Commiting with message \"{}\".", commit_msg);
```

Comment:
why println, if there's `info!` macro?

Reply:
The info macro prints useless timestamps into the console.

Reply:
It's easily tweakable using fern crate. We can leave info and create verbosity switcher to show messages with different priority.

Reply:
Ok I'll first merge this and implement that afterwards.